### ISSUE TITLE
Wrong color for haskell

### ISSUE THREAD

**Issue by adius** [github/linguist#3727]
The haskell community has obviously settled with purple (check out https://haskell.org, https://haskell-lang.org, https://reddit.com/r/haskell, ...). `rgb(94,80,134)` to be precise.
At the moment GitHub, however, uses green. This should be changed.

**Comment by Alhadis** [comment]
We like green. Go away.

Kidding. No, you're right, this should certainly be updated. Though [HaskellWiki](https://wiki.haskell.org/Brand#Palette) says `DarkBlue`, but from the looks of things, I'd say that's outdated.

**Comment by Alhadis** [comment]
All done! See #3728. If you're *very* lucky, this might even be included in the about-to-be-shipped release of Linguist (meaning it'll take effect on GitHub very shortly after approval).

**Comment by adius** [comment]
Nice! I was preparing for a day long argument 😂, but cool to see that decisions can actually be made quickly.
There just one thing to clarify: I just realized that it's actually a gradient between rgb(79, 67, 113) and rgb(110, 93, 157) and not a solid color.
So I'm not sure which color should actually be used...
I guess the brighter one?

**Comment by Alhadis** [comment]
Hold on, is this based on an official styleguide? You said _"`rgb(94,80,134)` **to be precise**"_, which lead me to believe this was being drawn from concrete branding guidelines...

Do you have a link to the *official* palette perchance?

**Comment by adius** [comment]
Sorry, I was maybe a little exuberant with my language. I was looking for a style guide, but there is none. The most official it gets ist the official logo used on haskell.org: https://github.com/haskell-infra/hl/blob/master/design/logo.svg
I just erroneously used a color picker instead of checking the source, which didn't work as expected as it had a gradient. 🙈

**Comment by Alhadis** [comment]
When looking for colour choices, I always start with the logo's palette. =)

It's clear that the colour needs to be changed... heck, even Haskell's boring old logo used purple!

![oldhaskelllogo](https://user-images.githubusercontent.com/2346707/28491029-b55ceebc-6f2b-11e7-97ee-dca1d7a47f94.png)

From the colours used in the [Haskell site's SVG logo](https://www.haskell.org/static/img/haskell-logo.svg), I'd say `#5e5086` (sampled from the centre lambda) is probably the best pick. Which is the same shade as the one you've chosen, so... no change necessary.

... there's also no gradient. :o

**Comment by adius** [comment]
> there's also no gradient

Now I'm really confused. The link I send you is supposed to be the source code for the haskell.org website. So it should be the same file as the link you posted, but obviously it's not 😳

... oh, I see. They have 2 logos in their source code:

- https://github.com/haskell-infra/hl/tree/master/static/img
- https://github.com/haskell-infra/hl/blob/master/design/logo.svg

and they are slightly different 😭 ... what a fail. I'll open a pull request.

Long story short let's stick with `#5e5086` 👍

**Comment by Alhadis** [comment]
I'm colourblind anyway, so everything that's been mentioned so far looks the same shade of purple to, lel.

**Comment by adius** [comment]
Unification of logos issue: https://github.com/haskell-infra/hl/issues/215

**Comment by Alhadis** [comment]
They won't do anything about it.

Haskell is lazy, remember?

<a href="https://www.youtube.com/watch?v=1u32Kr3sAoY"><img src="https://user-images.githubusercontent.com/2346707/28491218-7d2bfcd6-6f30-11e7-96dd-f3de8b59fe3e.png" alt="kek" /></a>

**Comment by chrisdone** [comment]
Cool! The theme was [my theme](http://chrisdone.com/posts/haskell-lang#the-theme), but based on Haskell's much older purple logo anyway.
